resources/definitions/anycubic_chiron.def.json

```diff
@@ -0,0 +1,80 @@
+{
+    "version": 2,
+    "name": "Anycubic Chiron",
+    "inherits": "fdmprinter",
+    "metadata":
+    {
+        "visible": true,
+        "author": "Patrick Glatt",
+        "manufacturer": "Anycubic",
+		"category": "Other",
+        "file_formats": "text/x-gcode",
+        "icon": "icon_ultimaker2",
+        "platform": "anycubic_chiron_platform.obj",
+		"platform_texture": "anycubic-chiron.png",
+        "has_materials": true,
+		"preferred_material": "generic_pla",
+        "has_machine_quality": true,
+		"quality_definition": "anycubic_chiron",
```

Comment:
Bit of nitpicking, but your indentation is a bit off.

Reply:
Y thx
Its not the same like it looks in notepad++. In notepad++ it was fine.
I will change this